### ISSUE TITLE
Forwarding a conversation should preserve structure

### ISSUE THREAD

**Issue by apecity** [freescout-helpdesk/freescout#3640]
Using Freescout 1.8.115 on Ubuntu.  PHP 8.1.12, Mysql 8.0.35.  Not using Cloudflare.

When forwarding a conversation, all the attachments from the conversation are grouped together at the end, and do not appear with the original messages.

This makes it difficult for the recipient to read a forwarded conversation, as all of the attachments are out of context.  

I'm not sure why Freescout is doing this, not preserving the structure of a forwarded conversation.  Could this be fixed?

If that it too tall of an order, could you put substitution links or something in the original messages that had the attachments, e.g. "See attachment named something.pdf at the end of this message."

Thank you!

**Comment by freescout-helpdesk** [comment]
> When forwarding a conversation, all the attachments from the conversation are grouped together at the end, and do not appear with the original messages.

This is how emails are working on the Internet - all attachments are attached to the email, there is no way to attach an attachment to some particular text in the email.

**Comment by apecity** [comment]
Hi.    

When you forward a conversation (aka message thread) from Outlook, GMail, etc., the attachments are displayed with the messages they were attached to.  Just as you said, "all attachments are attached to the email".  Great. Here is what a short conversation between a GMail user and Freescout looks like in GMail.  All good.
![chrome_668](https://github.com/freescout-helpdesk/freescout/assets/18216162/6d407cf5-6648-4098-845f-53d4789aa3b2)

Here is what the same conversation as shown above looks like in Freescout.  Again, all good. The message order is clear, and the attachments are displayed with the messages they were attached to.
![firefox_669](https://github.com/freescout-helpdesk/freescout/assets/18216162/9bb4ea4f-63ea-47d7-aa2f-cff8755e421d)

But now let's look at what this conversation looks like when I forward it from Freescout.
![chrome_670](https://github.com/freescout-helpdesk/freescout/assets/18216162/48b9016e-9a56-4498-8343-d96e7a3055e0)

The message order is still good, but all of the attachments have been pulled out and attached at the end.

That is what I mean by Freescout not preserving the structure - maybe I should have said preserve the conversation view.  I realize that when you forward a conversation in GMail, it does the exact same thing. As you said, it is an email, and the attachments are added to the email, not to some particular text.  

In a nutshell, I'd the conversation forwarded to the recipient in such a manner that it looks like it does in Freescout.  That is, it is readable, and the attachments appear with their original messages.  

Thank you,
Ted

**Comment by freescout-helpdesk** [comment]
Unfortunately we don't have ideas how technically it can be implemented. If someone does - please share or submit a pull request.
